Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -19,324 +19,26 @@ Create the desired types of storage in the provider Ceph cluster:
 * [RBD pools](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool)
 * [CephFS filesystem](https://docs.ceph.com/en/quincy/cephfs/createfs/)
 
-## Commands on the source Ceph cluster
+## Steps to connect external Provider cluster to Consumer cluster
 
-In order to configure an external Ceph cluster with Rook, we need to extract some information in order to connect to that cluster.
+* [Extract data from Provider Ceph cluster](/Documentation/CRDs/Cluster/external-cluster/provider-cluster.md)
```

Comment:
How about including a sentence or two with each of these links that describes the export action? Something like this? 
```suggestion
1. [Export config from the Provider Ceph cluster](/Documentation/CRDs/Cluster/external-cluster/provider-cluster.md). Configuration must be exported by the Ceph admin, such as a Ceph keyring and mon endpoints, that allows connection to the Ceph cluster.
```
Also, let's make this a numbered list so it is clear the steps are ordered.